packages/parser/src/actions/CombatAbsorbAction.ts

```diff
@@ -21,10 +21,10 @@ export class CombatAbsorbAction extends CombatAction {
   constructor(logLine: ILogLine, wowVersion: WowVersion) {
     super(logLine);
     if (!CombatAbsorbAction.supports(logLine)) {
-      throw new Error('event not supported');
+      throw new Error('Event not supported as CombatAbsorbAction');
```

Comment:
the error thrown from CombatAction.ts appends logLine.raw at the end. Should we consistently do so?

Reply:
yeah converting them all to a style that announces what event name failed and put the log line in there too